.devcontainer/post-start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ -x "$(command -v docker-compose)" ]; then
+  docker-compose up --detach
+else
+  docker compose up --detach
+fi
```

Comment:
Is this redundant since it's in the post-create script?

Reply:
At first I had this only in `post-start.sh` because it is needed to start `docker` each time the machine starts, but I forgotten that I need to start once `npm run migrate:dev:db`. Without this script application isn't working correctly.

Script `post-create.sh` is executed only once, when the machine is created and it the right place to execute `npm run migrate:dev:db`, but to run this the database needs to be started and so `docker compose up --detach` is required.
Maybe it would be better to call `sh ./post-start.sh` instead of this `if`.

I'm not sure, if there should be another check if docker isn't running. Because on the first time `post-create.sh` is executed and then `post-start.sh` (two scripts, one after other). And then (everyother start of the machine) only `post-start.sh`. But at the first time (when machine is created) `docker compose up` is called two times. Not sure if this ends with error which of course can be ignored but is not nice. Let me check this...

Reply:
Running multiple times `docker compose up` doesn't throws any errors it just displays that everything is running, then it is better to leave it this way, without checking it in the shell script.

In summary this block in `post-create.sh` script can be replaced with call to `sh ./post-start.sh` script, but I'm not sure what is the working directory, then it would be better to call it this way:
```shell
# Get script directory
DIR="$(cd "$(dirname "$0")" && pwd)"

# Start Docker before running: npm run migrate:dev:db
sh "$DIR/post-start.sh"
```
The `DIR` variable can be set at the start of `post-create.sh` script, because it could be used in fututre in other places (?).